server/plugin.go

```diff
@@ -138,6 +156,123 @@ func (p *Plugin) handleList(w http.ResponseWriter, r *http.Request) {
 	w.Write(itemsJSON)
 }
 
+type enqueueAPIRequest struct {
```

Comment:
2/5 I _really_ don't like the term `Enqueue` here, can we express it with the existing vocabulary?

Reply:
What do you think about "Accept"? Since it is used to "accept as my own to do something I have been sent", I think it should work.